chore: move production environment into its own page (#314)
openfga/openfga.dev: docs/content/modeling/migrating/migrating-schema-1-1.mdx

      link: './migrating-relations',
      id: './migrating-relations.mdx',
    },
    {
      title: 'Running OpenFGA in Production Environment',

>       description: 'Learn about how to run OpenFGA in a production environment.',

Reviewer: ```suggestion
      description: 'Learn how to run OpenFGA in a production environment',
```

>       link: '../../getting-started/production-environment',

Reviewer 2: ```suggestion
      link: '../../getting-started/running-in-production',
```

>       id: '../../getting-started/production-environment',

Reviewer 2: ```suggestion
      id: '../../getting-started/running-in-production',
```
